measurement/measures/volumetric_flow.py

```diff
@@ -0,0 +1,41 @@
+from measurement.base import BidimensionalMeasure
+
+
+from measurement.measures.volume import Volume
+from measurement.measures.time import Time
+
+
+__all__ = [
+    'VolumetricFlow'
+]
+
+
+class VolumetricFlow(BidimensionalMeasure):
+    """ Volumetric Flow measurements (generally for water flow). """
```

Comment:
Please remove the padding whitespaces.
```suggestion
    """Volumetric Flow measurements (generally for water flow)."""
```